import-scripts/cvr_consent_status_checker.py

```diff
@@ -192,6 +197,16 @@ def main():
         sys.exit(2)
 
     expected_consent_status_values = fetch_expected_consent_status_values()
+    if expected_consent_status_values is None:
```

Comment:
None is too generic and doesn't cover the fact that PARTA might be working but PARTC might not - see above comment about preserving struct and then doing a check inside the main